### DISCUSSION TITLE
xarray dataset or dataarray html display failed in jupyter notebook

### DISCUSSION THREAD

**Discussion by Jing25** [pydata/xarray#5974] [Q&A] [answered]
I cannot have xarray dataset or dataarray displayed in jupyter notebook (I'm actually using jupyterhub). It shows blank.

<img width="283" alt="image" src="https://user-images.githubusercontent.com/27107288/141218794-ac3a1172-f83f-4a94-b3c9-3b9aec1ba071.png">

'text' style works: 
<img width="630" alt="image" src="https://user-images.githubusercontent.com/27107288/141218759-c4d339cd-e615-42d8-a756-d095cb37e2a3.png">

I checked my jupyter notebook HTML display and it works great:
<img width="383" alt="image" src="https://user-images.githubusercontent.com/27107288/141218972-792df902-4abf-4c60-b844-bc5f3b12431e.png">

No problem with displaying pandas dataframe also. 

So I excluded jupyter notebook as the source of the issue. 

I'm not sure if this is a bug of xarray but does anyone know how I can solve this problem?

My xarray version is 0.19.0, Ubuntu 20.04.3.

**Comment by andersy005** [comment, marked as answer]
Do you have any other jupyterlab extensions installed in your environment? Your issue seems to be related to  
- https://github.com/executablebooks/sphinx-book-theme/issues/238 
- https://github.com/pydata/xarray/pull/5880#issue-1031724989

**Reply by Jing25** [reply]
Yes, exactly! I deleted "hidden" as is in https://github.com/executablebooks/sphinx-book-theme/issues/238 and the content showed up. I can't remember if I installed any other extensions. Has this been solved?
